Makefile

```diff
@@ -1,6 +1,13 @@
 TARGET = lfs
 
-CC = gcc
+OS := $(shell uname -s)
+
+ifeq ($(OS), FreeBSD)
+	CC = cc
```

Comment:
I believe `cc` is also available on Linux, we should actually be using that alias for both OSs.

Reply:
Yes, that would be more appropriate in my opinion also.
On my VM build pass when <code>CC = cc</code> is set.